### ISSUE TITLE
Stream Scoring: ImportError: cannot import name 'easter' from 'holidays'

### ISSUE THREAD

**Issue by lehmus** [microsoft/MCW-Machine-Learning#16]
In Exercise 3 (Stream Scoring notebook), the above error appears in cell 13 (experiment run id obfuscated afterwards):

![Error](https://user-images.githubusercontent.com/16114036/73642789-02326400-467b-11ea-8c15-a3927ade8cd2.png)

The error is caused by this line:
`best_run, best_model = automl_run.get_output()`

The same notebook was able to run successully a few weeks ago, but now it crashes. I instructed this workshop on Friday 31 January 2020, and all the participants were able to reproduce the issue. Here is the full stack trace from notebook output:

```
ImportError                               Traceback (most recent call last)
<command-708075521530493> in <module>
      5 
      6 automl_run = AutoMLRun(existing_experiment, run_id)
----> 7 best_run, best_model = automl_run.get_output()

/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/azureml/train/automl/run.py in get_output(self, iteration, metric, return_onnx_model, return_split_onnx_model, **kwargs)
    493             )
    494         else:
--> 495             fitted_model = self._download_automl_model(curr_run)
    496 
    497         return curr_run, fitted_model

/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/azureml/train/automl/run.py in _download_automl_model(self, curr_run)
    605             import azureml.train.automl.runtime
    606             with open(self.local_model_path, "rb") as model_file:
--> 607                 fitted_model = pickle.load(model_file)
    608         except ImportError as e:
    609             # Check to see if importing azureml.train.automl.runtime specifically failed

/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/fbprophet/__init__.py in <module>
      6 # of patent rights can be found in the PATENTS file in the same directory.
      7 
----> 8 from fbprophet.forecaster import Prophet
      9 
     10 __version__ = '0.5'

/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/fbprophet/forecaster.py in <module>
     18 
     19 from fbprophet.diagnostics import prophet_copy
---> 20 from fbprophet.make_holidays import get_holiday_names, make_holidays_df
     21 from fbprophet.models import prophet_stan_model
     22 from fbprophet.plot import (plot, plot_components, plot_forecast_component,

/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/fbprophet/make_holidays.py in <module>
     14 import pandas as pd
     15 
---> 16 import fbprophet.hdays as hdays_part2
     17 import holidays as hdays_part1
     18 

/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/fbprophet/hdays.py in <module>
     14 
     15 from convertdate.islamic import from_gregorian, to_gregorian
---> 16 from holidays import WEEKEND, HolidayBase, easter, rd
     17 from lunardate import LunarDate
     18 

ImportError: cannot import name 'easter' from 'holidays' (/local_disk0/pythonVirtualEnvDirs/virtualEnv-ce55e929-5c7f-400c-bcae-bb5fde908b92/lib/python3.7/site-packages/holidays/__init__.py)```

**Comment by shirolkar** [comment]
@lehmus @timahenning Thanks for reporting this issue. I will provide an update soon. Jitendra

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #17
